pkgs/development/tools/metal-cli/default.nix

```diff
@@ -2,21 +2,24 @@
 
 buildGoModule rec {
   pname = "metal-cli";
-  version = "0.7.3";
+  version = "0.7.4";
 
   src = fetchFromGitHub {
     owner = "equinix";
     repo = pname;
     rev = "v${version}";
-    sha256 = "sha256-PsTQuEbo1ReWwZV4D14mEGsU99j49kleiL/6Xyk+g9s=";
+    sha256 = "sha256-muhHBUb5Ttj4n6fJzIJMqics5rKupeSBZAd4JxZUe64=";
   };
 
+  vendorSha256 = "sha256-F8d5i9jvjY11Pv6w0ZXI3jr0Wix++B/w9oRTuJGpQfE=";
+
   ldflags = [
     "-X github.com/equinix/metal-cli/cmd.Version=${version}"
   ];
 
-  vendorSha256 = "sha256-F8d5i9jvjY11Pv6w0ZXI3jr0Wix++B/w9oRTuJGpQfE=";
-
+  # FIX-ME: Tests exist but aren't running.
+  #   go build github.com/equinix/metal-cli/tests: no non-test Go files in /build/source/tests
+  #   could not build binary for metal: exit status 1FAIL     github.com/equinix/metal-cli/tests
   doCheck = false;
```

Comment:
Also, overriding `checkPhase` does not help as the test itself tries to build the binary (per https://github.com/equinix/metal-cli/blob/main/tests/main_test.go#L24).

I did try:

```
  checkPhase = ''
      runHook preCheck
      buildGoDir test $checkFlags ./tests
      runHook postCheck
    '';
```

But:

```
running tests
build
could not build binary for metal: exit status 1FAIL     github.com/equinix/metal-cli/tests      0.022s
FAIL
```

I do no think that a workarround is of value here, as we will basically redo the tests, you can just add a installCheckPhase.

Reply:
Done